src/viewport.js

```diff
@@ -1125,17 +1124,18 @@ $.Viewport.prototype = {
         return changed;
     },
 
-    // private
-    _rotateAboutPivot: function(immediately, degrees){
-        //immediately defaults to false; degrees are ignored for immediately==false
+    // private - pass true to use spring, or a number for degrees for immediate rotation
+    _rotateAboutPivot: function(degreesOrUseSpring){
```

Comment:
I'm fine with this API as well if you think it's better than `immediateDegrees` :)